webapp/niapp.yaml

```diff
@@ -235,7 +255,32 @@ paths:
           $ref: '#/responses/Unauthorized'
         404:
           $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
 
+  '/webapps/duplicate/{sourceId}':
+    post:
+      tags:
+        - content
+      summary: Duplicate webapp
+      description: Duplicate a webapp by creating new webapp metadata with the specified name and duplicating existing content into the new webapp.
+      operationId: duplicate-webapp
+      parameters:
+        - in: path
+          type: string
+          required: true
+          name: sourceId
+          description: The Id of the source webapp to copy content from
+        - $ref: '#/parameters/DuplicateWebAppRequest'
+      responses:
+        200:
+          description: Success
```

Comment:
A successful request should include the newly-created webapp's metadata in the response right? I would expect the documentation to outline a response in the success case similar to the way the successful response is described for the create route.